earth_enterprise/src/common/serverdb/serverdbUtilities_unittest.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Open GEE Contributors
+// Unit tests for ServerdbUtilities.
+
+#include <gtest/gtest.h>
+#include <string>
+#include "common/serverdb/serverdbUtilities.h"
+
+
+class ServerdbUtilitiesTest : public testing::Test {
+protected:
+    virtual void SetUp() {}
+
+    virtual void TearDown() {
+    }
+
+    std::string GetJavaScriptString() {
+        std::string test_string =
+        "{"
+        "dbType : \"gedb\","
+        "isAuthenticated : false,"
+        "layers :"
+        "["
+        "{"
+        "description : \"\","
+        "id : \"4c8ca722-73cd-11e8-8e36-a969646ae0ac\","
+        "isEnabled : true,"
+        "isExpandable : false,"
+        "isInitiallyOn : true,"
+        "isVisible : true,"
+        "label : \"Imagery\","
+        "lookAt : \"none\""
+        "}"
+        "]"
+        ","
+        "serverUrl : \"http://localhost/testdb-v001\""
+        "}";
+        return test_string;
+    }
+
+    std::string GetJSONString() {
+        std::string json_string =
+        "{"
+        "\"dbType\" : \"gedb\","
```

Comment:
Although this is valid JSON, some parsers don't handle white-space in JSON properly.  I don't know what clients we're trying to support, but I think we'd get wider compatibility, if we use minified JSON, i.e. with no white-space.